bears/css/StyleLintBear.py

```diff
@@ -0,0 +1,32 @@
+import os
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylelint',
+        output_format='regex',
+        output_regex=r'.+\n\s(?P<line>\d+):(?P<column>\d+)'
+                     r'\s+✖\s(?P<message>.+)')
```

Comment:
@aptrishu I'll add more test cases... however I can't figure out what changes I need to make in `bear-requirements.txt` and `.ci/deps.sh`... can you help me out with these ?